code/drasil-example/pdcontroller/lib/Drasil/PDController/Unitals.hs

```diff
@@ -2,12 +2,18 @@ module Drasil.PDController.Unitals where
 
 import Data.Drasil.Constraints (gtZeroConstr)
 import Data.Drasil.SI_Units (second)
+import Data.Drasil.Quantities.Physics (vecDim)
 import Language.Drasil
+import Language.Drasil.Space (ClifKind(..))
 import Language.Drasil.Chunk.Concept.NamedCombinators
 
 import Drasil.PDController.Concepts
 import Control.Lens ((^.))
 
+-- | Helper function to create Clifford vector spaces of a given dimension
+realVect :: Dimension -> Space
```

Comment:
This doesn't belong in an example, it should be abstracted out. I'd say in `drasil-lang` for now.